### ISSUE TITLE
Laravel PHP - File cannot be parsed if a string contains a single quote

### ISSUE THREAD

**Issue by guyaumetremblay** [mirego/accent#380]
Hi, 

Using a Laravel PHP file, if a string (double quoted) contains a single quote, the cli cannot parse the file. For example: 

```
<?php

return [
    'foo' => [
        'bar' => [
            'cta' => "Confirmer à l'équipe"
        ]
    ]
];
```

With this file, the cli return this message: 
```
 ›   Error: file cannot be parsed
```

Single quoted string, escaping the single quote like this, doesn't work either:  
```
<?php

return [
    'foo' => [
        'bar' => [
            'cta' => 'Confirmer à l\'équipe'
        ]
    ]
];
```

We are using both across our laravel projects. 

Thanks for your help!

**Comment by guyaumetremblay** [comment]
@simonprev Should I open the issue in https://github.com/nicklayb/php_assoc_map ? 
cc @nicklayb

**Comment by nicklayb** [comment]
I don't remember if I updated accent to use to lexer/parser version of pho_assoc_map, I'd try that before. It's way more reliable than the tokenizer-ish version that was in place before. But I can sure look into it, open an issue on the pho_assoc_map lib with your failing example

**Comment by guyaumetremblay** [comment]
@nicklayb Sorry for the delay! I opened the issue https://github.com/nicklayb/php_assoc_map/issues/9

**Comment by guyaumetremblay** [comment]
Fixed in #389 !
